packages/components/src/components/div100vh-container/div100vh-container.tsx

```diff
@@ -14,14 +14,15 @@ import Div100vh from 'react-div-100vh';
 /* To manually remove rvh calculation and revert to default browser calculation use is_disabled */
 /* To bypass usage of component altogether, use is_bypassed */
 
-type TDiv100vhContainer = {
-    id: string;
-    height_offset: string;
-    is_bypassed: boolean;
-    is_disabled: boolean;
-    max_height_offset: string;
-    className: string;
-    max_autoheight_offset: string;
+type TDiv100vhContainerProps = {
+    id?: string;
+    height_offset?: string;
+    is_bypassed?: boolean;
+    is_disabled?: boolean;
+    max_height_offset?: string;
+    className?: string;
+    max_autoheight_offset?: string;
+    children: React.ReactNode;
```

Comment:
Can we not provide children prop explicitly and wrap the type with `React.PropsWithchildren` ?